CMakeLists.txt

```diff
@@ -18,11 +18,10 @@ project(FreeCAD)
 set(PACKAGE_VERSION_NAME "Vulcan")
 set(PACKAGE_VERSION_MAJOR "0")
 set(PACKAGE_VERSION_MINOR "21")
-set(PACKAGE_VERSION_PATCH "0")
+set(PACKAGE_VERSION_PATCH "0") # number of patch release (e.g. "4" for the 0.18.4 release)
 set(PACKAGE_VERSION_SUFFIX "dev") # either "dev" for development snapshot or "" (empty string)
-set(FREECAD_VERSION_PATCH "0") # number of patch release (e.g. "4" for the 0.18.4 release)
```

Comment:
does somebody know if this is something that may be used by other programs that link freecad as a library?

Reply:
I just checked today around because I had the same idea as you and wondered why there is FREECAD_VERSION_PATCH and PACKAGE_VERSION_PATCH.

I found no reason,, so let's give it a try. However, to be safe, not for the 0.20 branch.